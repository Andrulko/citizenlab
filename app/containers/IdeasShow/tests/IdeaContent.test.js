describe('<IdeaContent />', () => {
  it('should render idea\'s multiloc content', () => {
    // TODO
    expect(true).toEqual(true);
  });
});
