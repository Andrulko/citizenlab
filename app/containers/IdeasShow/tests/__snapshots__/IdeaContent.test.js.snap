// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IdeaContent /> should render idea's multiloc content 1`] = `
<div
  className="foqgVb"
>
  <span>
    some content
     
  </span>
  <hr />
</div>
`;
